src/components/UI/atoms/InputText/styles.module.css

```diff
@@ -0,0 +1,32 @@
+.Form__Label {
+  width: 50%;
```

Comment:
I wouldn't recommend specifying a `width` here as this is a reusable atom and in the future it could be used somewhere that width doesn't work. I would rather delegate the responsibility of establishing that `width` to a parent element.